src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1726,6 +1726,41 @@ properties:
             title: Observatory UTC time at guider function end
             type: string
             fits_keyword: GF_END
+          fgs_aperture:
+            title: Aperture used for guide star acquisition
+            type: string
+            fits_keyword: FGS_APER
+            blend_table: True
+          gs_ideal_x:
+            title: Guide star ideal y coordinate
+            type: number
+            fits_keyword: GSIDL_X
+            blend_table: True
+          gs_ideal_y:
+            title: Guide star ideal y coordinate
+            type: number
+            fits_keyword: GSIDL_Y
+            blend_table: True
+          fgs_v2_ref:
+            title: "[arcsec] Telescope V2 coord of reference point"
+            type: number
+            fits_keyword: FGSV2REF
+            blend_table: True
+          fgs_v3_ref:
+            title: "[arcsec] Telescope V3 coord of reference point"
+            type: number
+            fits_keyword: FGSV3REF
+            blend_table: True
+          fgs_v3yangle:
+              title: "[deg] Angle from V3 axis to Ideal y axis"
+              type: number
+              fits_keyword: FGSV3YAN
+              blend_table: True
+          fgs_vparity:
+            title: Relative sense of rotation between Ideal xy and V2V3
+            type: integer
+            fits_keyword: FGS_VPAR
+            blend_table: True
```

Comment:
These four entries don't match the keywords that SDP will populate - they are using V2_REF, V3_REF, V3I_YANG and VPARITY. Because these match the keyword entries in `wcsinfo` for SCI extensions, it may be useful to specify the fits_hdu for these entries as PRIMARY. I don't have great intuition on if this will cause issues, but this would match the update made in the grit merge request filed by Linda against JWSTKD-436. We can see if RTs find problems.

Reply:
They match what's in the Excel spreadsheet for FITS keyword... for future reference how do I find what is _actually_ needed myself?  It sounds like you looked at the grit merge request?  When I try to follow the link I get a 404 page not found

Reply:
ok yeah I agree it's a slightly different case and probably the linked issue should not be relevant.
how does https://github.com/spacetelescope/stdatamodels/pull/500/commits/519373c717b7886254d538caed8bc86f9ac78d86 look to you?